USAGE.md

```diff
@@ -0,0 +1,260 @@
+
+- `connect.NewClient` – Accepts a hostname and a token value.
+
+  ```go
+  package main
+
+  import "github.com/1Password/connect-sdk-go/connect"
+
+  func main () {
+      client := connect.NewClient("<your_connect_host>", "<your_connect_token>")
+  }
+  ```
+
+- `connect.NewClientFromEnvironment` – Fetches the hostname and token value from the environment, and expects these to be passed as environment variables (`OP_CONNECT_HOST` and `OP_CONNECT_TOKEN`, respectively).
+
+Assuming that `OP_CONNECT_TOKEN` and `OP_CONNECT_HOST` have been set as environment variables, the `connect.NewClientFromEnvironment` can be invoked as such:
+
+```go
+package main
+
+import "github.com/1Password/connect-sdk-go/connect"
+
+func main () {
+	client, err:= connect.NewClientFromEnvironment()
+	if err != nil {
+		panic(err)
+	}
+}
+```
+
+- `connect.NewClientWithUserAgent` – Accepts a hostname, a token value, and a custom User-Agent string for identifying the client to the 1Password Connect API:
+
+```go
+package main
+
+import "github.com/1Password/connect-sdk-go/connect"
+
+func main () {
+	client := connect.NewClientWithUserAgent("<your_connect_host>", "<your_connect_token>", "Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_2 like Mac OS X) AppleWebKit/603.2.4 (KHTML, like Gecko) FxiOS/8.1.1b4948 Mobile/14F89 Safari/603.2.4")
+}
+```
+
+## Working with Vaults
+
+```go
+// Get a list of all vaults
+vaults, err := client.GetVaults()
+if err != nil {
+	log.Fatal(err)
+}
+
+// Get a specific vault
+vault, err := client.GetVault("vaultID _or_ vaultTitle")
+if err != nil {
+	log.Fatal(err)
+}
+
+vaultByID, err := client.GetVaultById("valutID")
+if err != nil {
+    log.Fatal(err)
+}
+
+vaultByTitle, err := client.GetVaultByTitle("valutTitle")
+if err != nil {
+    log.Fatal(err)
+}
+```
+
+## Working with Items
+
+### CRUD operations on items
+
+```go
+vault := "vaultID _or_ vaultTitle"
+
+// Get a list of all items in a vault
+items, err := client.GetItems(vault)
+if err != nil {
+	log.Fatal(err)
+}
+
+// Create an item
+newItem := &onepassword.Item{
+	Title:    "Secret String",
+	Category: onepassword.Login,
+	Tags:     []string{"1password-connect"},
+	Fields: []*onepassword.ItemField{{
+		Value: "mysecret",
+		Type: "STRING",
+	}},
+}
+
+createdItem, err := client.CreateItem(newItem, vault)
+if err != nil {
+	log.Fatal(err)
+}
+
+// Get an item
+item, err := client.GetItem("itemID _or_ itemTitle", vault)
+if err != nil {
+	log.Fatal(err)
+}
+
+itemByID, err := client.GetItem("itemID", vault)
+if err != nil {
+	log.Fatal(err)
+}
+
+itemByTitle, err := client.GetItem("itemTitle", vault)
+if err != nil {
+	log.Fatal(err)
+}
+
+// Update na item
+item.Title = "New Item Title"
+updatedItem, err := client.UpdateItem(item, vault)
+if err != nil {
+	log.Fatal(err)
+}
+
+// Delete an item
+err = client.DeleteItem(item, vault)
+if err != nil {
+    log.Fatal(err)
```

Comment:
Two extra spaces of indentation here can be removed.

```suggestion
  log.Fatal(err)
```

Reply:
There was a formatting issue here compared to the rest of the code. It seems that it used 4 spaces instead of a tab character for this line (and some others), which led to the weird misalignment. Now it should be fixed.